lib/theme/app_theme.dart

```diff
@@ -5,17 +5,31 @@ class AppTheme {
   static ThemeData get light => ThemeData(
         fontFamily: FontFamily.neuzeit,
         textTheme: const TextTheme(
-          bodyMedium: TextStyle(
-            color: Colors.white,
-            fontSize: 17,
-            fontWeight: FontWeight.normal,
-          ),
-          labelMedium: TextStyle(
-            color: Colors.white,
-            fontSize: 17,
-            fontWeight: FontWeight.bold,
-          ),
-        ),
+            bodyMedium: TextStyle(
+              color: Colors.white,
+              fontSize: 17,
+              fontWeight: FontWeight.normal,
+            ),
+            labelMedium: TextStyle(
+              color: Colors.white,
+              fontSize: 17,
+              fontWeight: FontWeight.bold,
+            ),
+            labelSmall: TextStyle(
+              color: Colors.white,
+              fontSize: 13,
+              fontWeight: FontWeight.w800,
+            ),
+            labelLarge: TextStyle(
+              color: Colors.white,
+              fontSize: 34,
+              fontWeight: FontWeight.w800,
+            ),
+            titleLarge: TextStyle(
+              color: Colors.white,
+              fontSize: 28,
+              fontWeight: FontWeight.w800,
```

Comment:
- We can use `FontWeight.bold` instead of `FontWeight.w800`.
- If I'm not wrong, the font sizes decrease in the order: `display` > `headline` > `title` > `body` > `label`. In this case, the `34` size would be `titleLarge`, and the `28` size would be `titleMedium`.